### ISSUE TITLE
PHP Warning:  Undefined array key "HTTP_ACCEPT_LANGUAGE" in Contao/Environment.php on line 229"

### ISSUE THREAD

**Issue by BugBuster1701** [contao/contao#5514]
### Affected version(s)

4.13.13

### Description

I find regular entries with it in my error.log
```
[17-Nov-2022 23:17:59] WARNING: [pool www] child 15628 said into stderr: "[17-Nov-2022 23:17:59 Europe/Berlin] PHP Warning:  Undefined array key "HTTP_ACCEPT_LANGUAGE" in /var/www/x/vendor/contao/core-bundle/src/Resources/contao/library/Contao/Environment.php on line 229"
[17-Nov-2022 23:53:39] WARNING: [pool www] child 26174 said into stderr: "[17-Nov-2022 23:53:39 Europe/Berlin] PHP Warning:  Undefined array key "HTTP_ACCEPT_LANGUAGE" in /var/www/x/vendor/contao/core-bundle/src/Resources/contao/library/Contao/Environment.php on line 229"
```

**Comment by ausi** [comment]
We should probably use `$_SERVER['HTTP_ACCEPT_LANGUAGE'] ?? null` here: https://github.com/contao/contao/blob/f8c6ca565e92311bac1cb71a50e5c5ce92674237/core-bundle/src/Resources/contao/library/Contao/Environment.php#L229

**Comment by aschempp** [comment]
see https://github.com/contao/contao/pull/5517
FYI I reassigned this from 4.9 to 4.13 since we only fix these PHP8 issues in 4.13

**Comment by leofeyer** [comment]
> FYI I reassigned this from 4.9 to 4.13 since we only fix these PHP8 issues in 4.13

Since when is that?
